node/src/indexer.rs

```diff
@@ -91,7 +91,7 @@ async fn handle_block(
                             tracing::warn!("`sign` did not produce entropy");
                             continue;
```

Comment:
should this check len() > 1 instead of empty?

Reply:
not sure what you mean here. can you please elaborate?